docs/extensions-integrations/integration-guides/calling-commands.md

```diff
@@ -25,13 +95,71 @@ public class ExampleIntegration  {
         // ... setup configuration system and file access
 
         try {
-            new CommandScope("update")
-                    .addArgumentValue("changeLogFile", "my/changelog.xml")
-                    .execute();
+            CommandScope updateCommand = new CommandScope("update");
+            updateCommand.addArgumentValue(UpdateCommandStep.CHANGELOG_FILE_ARG, changeLogFile);
+            updateCommand.addArgumentValue(UpdateCommandStep.CONTEXTS_ARG, contexts);
+            updateCommand.addArgumentValue(UpdateCommandStep.LABEL_FILTER_ARG, labelFilter);
+            updateCommand.addArgumentValue(ChangeExecListenerCommandStep.CHANGE_EXEC_LISTENER_CLASS_ARG, changeExecListenerClass);
+            updateCommand.addArgumentValue(ChangeExecListenerCommandStep.CHANGE_EXEC_LISTENER_PROPERTIES_FILE_ARG, changeExecListenerPropertiesFile);
+            setDatabaseArgumentsToCommand(updateCommand);
+            updateCommand.execute();
         } catch (CommandExecutionException e) {
             System.out.println("Error running update: "+e.getMessage());
         }
+    }
+}
+```
+
+### Execution in scope example
```

Comment:
Why would I execute a command in a scope? What can I do with a scope?

Reply:
I thought this way and `Scope.child()` are at the end a similar thing so we can run any of these ways and set different scope settings. For example, a resource accessor, an env. variable, etc.

Reply:
Sorry, I mean... we are using the Scope here but there is no explanation why we are doing that . I think we would need a brief description of what is a Scope and how to use it before starting adding examples using that.